tools/api_proto_breaking_change_detector/BUILD

```diff
@@ -0,0 +1,67 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_test")
+
+licenses(["notice"])  # Apache 2
+
+py_binary(
+    name = "proto_breaking_change_detector",
+    srcs = ["detector.py"],
+    data = [
+        "//tools/api_proto_breaking_change_detector:buf.lock",
+        "//tools/api_proto_breaking_change_detector:buf.yaml",
```

Comment:
I think these can be listed just as `:buf.lock` and `:buf.yaml` instead of their fully-qualified names.